cmd/skr/main.go

```diff
@@ -108,6 +109,21 @@ func main() {
 		}
 	}
 
+	if common.ThimCertsAbsent(&EncodedUvmInformation.InitialCerts) {
+		logrus.Info("ThimCerts is absent, retrieving THIMCerts.")
+		thimCerts, err := info.CertFetcher.GetThimCerts("")
+		if err != nil {
+			logrus.Fatalf("Failed to retrieve thim certs: %s", err.Error())
+		}
+
+		EncodedUvmInformation.InitialCerts = *thimCerts
+	}
+
+	//temporary code. Remove uvm reference info if the running env is aks.
+	if msi.WorkloadIdentityEnabled() {
+		EncodedUvmInformation.EncodedUvmReferenceInfo = ""
+	}

```

Comment:
I also don't want to hard remove uvm info here.  Either it exists or it doesn't.  For testing purposes until we have production signed uvm info, just set the path to the uvm info to an invalid path and don't fail.